administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -148,6 +155,19 @@ const ApplicationCard = ({
     return !!blueCardJuleicaEntitlement
   }
 
+  const RightElement = (): ReactElement => {
+    return (
+      <RightElementContainer>
+        {application.note && <Icon icon='annotation' intent='none' />}
```

Comment:
```suggestion
        {application.note && application.note.trim() && <Icon icon='annotation' intent='none' />}
```
as a nit-picky corner case, we could also check that the note is not just a space.
the use-case - if the user wanted to remove the note but accidentally left a space for example.